x-pack/legacy/plugins/integrations_manager/server/integrations/handlers.ts

```diff
@@ -4,10 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Request } from '../../common/types';
-import { ArchiveEntry, pathParts } from '../registry';
+import { Request, ResponseToolkit } from '../../common/types';
+import { PluginContext } from '../plugin';
 import { getClient } from '../saved_objects';
-import { getIntegrations, getIntegrationInfo, installAssets, removeInstallation } from './data';
+import {
+  getClusterAccessor,
+  getIntegrationInfo,
+  getIntegrations,
+  installIntegration,
+  removeInstallation,
+} from './index';
+
+interface Extra extends ResponseToolkit {
```

Comment:
No idea what to call this type for the second arg to handlers `h`. Even the hapi docs kinda 🤷‍♀ 

> The response toolkit is a collection of properties and utilities passed to every lifecycle method. It is somewhat hard to define as it provides both utilities for manipulating responses as well as other information. Since the toolkit is passed as a function argument, developers can name it whatever they want. For the purpose of this document the h notation is used. It is named in the spirit of the RethinkDB r method, with h for hapi.
— https://hapijs.com/api#response-toolkit